ansible.cfg

```diff
@@ -13,3 +13,9 @@ callback_whitelist = profile_tasks
 roles_path = roles:$VIRTUAL_ENV/usr/local/share/kubespray/roles:$VIRTUAL_ENV/usr/local/share/ansible/roles:/usr/share/kubespray/roles
 deprecation_warnings=False
 inventory_ignore_extensions = ~, .orig, .bak, .ini, .cfg, .retry, .pyc, .pyo, .creds
+
+# uncoment for AWS if you have standart users like centos or ubuntu
```

Comment:
i think it's not just AWS specific, so probably it can be enabled by default? @mattymo what you think ? then on cmdline you should not provide additional -b